lexicons/app/bsky/notification/defs.json

```diff
@@ -8,17 +8,17 @@
     },
     "chatPreference": {
       "type": "object",
-      "required": ["filter", "push"],
+      "required": ["include", "push"],
       "properties": {
-        "filter": { "type": "string", "knownValues": ["all", "accepted"] },
+        "include": { "type": "string", "knownValues": ["all", "accepted"] },
         "push": { "type": "boolean" }
       }
     },
     "filterablePreference": {
```

Comment:
Think we should change this name, now that we're not saying that it applies a "filter"?